src/v2/Apps/Partner/Routes/Works/index.tsx

```diff
@@ -1,5 +1,60 @@
 import React from "react"
+import { createRefetchContainer, RelayRefetchProp } from "react-relay"
+import { graphql } from "relay-runtime"
+import { Articles_partner } from "v2/__generated__/Articles_partner.graphql"
+import { ArtworkFilterContextProvider } from "v2/Components/v2/ArtworkFilter/ArtworkFilterContext"
+import { BaseArtworkFilter } from "v2/Components/v2/ArtworkFilter"
+import { useRouter } from "v2/Artsy/Router/useRouter"
 
-export const WorksRoute: React.FC = () => {
-  return <div>Works</div>
+interface PartnerArtworkFilterProps {
+  partner: Articles_partner
+  relay: RelayRefetchProp
 }
+
+export const Artworks: React.FC<PartnerArtworkFilterProps> = ({
+  partner,
+  relay,
+}) => {
+  const { match } = useRouter()
+
+  return (
+    <ArtworkFilterContextProvider
+      filters={match && match.location.query}
+      sortOptions={[
+        { text: "Default", value: "-decayed_merch" },
+        { text: "Price (desc.)", value: "-has_price,-prices" },
+        { text: "Price (asc.)", value: "-has_price,prices" },
+        { text: "Recently updated", value: "-partner_updated_at" },
+        { text: "Recently added", value: "-published_at" },
+        { text: "Artwork year (desc.)", value: "-year" },
+        { text: "Artwork year (asc.)", value: "year" },
+      ]}
+    >
+      <BaseArtworkFilter relay={relay} viewer={partner as any} />
+    </ArtworkFilterContextProvider>
+  )
+}
+
+export const ArtworksRefetchContainer = createRefetchContainer(
+  Artworks,
+  {
+    partner: graphql`
+      fragment Works_partner on Partner
+        @argumentDefinitions(input: { type: "FilterArtworksInput" }) {
+        slug
+        internalID
+        filtered_artworks: filterArtworksConnection(first: 30, input: $input) {
+          id
+          ...ArtworkFilterArtworkGrid_filtered_artworks
+        }
+      }
+    `,
+  },
+  graphql`
+    query WorksQuery($partnerId: String!, $input: FilterArtworksInput) {
```

Comment:
#minor I might rename this to be a bit more scoped: "PartnerProfileWorksQuery", _maybe_ even similar for the fragment up top? All of these names (queries + fragment names) exist in a global namespace for the Force app, so "WorksQuery" is a little too generic IMO.